### ISSUE TITLE
Prep BAGTS presentation

### ISSUE THREAD

**Issue by lippytak** [codeforamerica/health#8]
It's a diverse audience (mostly startup focused but not many VCs on attendee list) so I think the goal is just to engage people and offer a pretty open-ended door for interested people to contribute. I could see audience members potentially being users, helping with user testing, contributing code, connecting with funders, etc.

Outline:
- Work at CfA building digital services to improve health (mostly in social services right now)
- Finding so far that the problems are more interesting than the solutions. So instead of a product demo i'm going to start with a few **problem demo**.
- Context = SNAP, Alan, enrolling, dogfooding
- Step 1 = Apply > MBCW
- Step 2 = Where to use my benefits > CA EBT
- Step 3 = Check my balance > Audio
- Step 4 = Call with questions > Audio
- Small problems add up = summarize cumulative impact
- So we're trying to build products to match these problems. Right now they're more like experiments.
- Clean > EBT > Balance > Connect
- Team
  - Right now we're a team of THREE currently building and supporting FOUR products for social service clients. Eep!
  - It's been really interesting but really hard. Checking your Food Stamps balance won't go viral, so we're figure out new growth strategies.
  - UserTesting.com doesn't hire social service clients as testers, so we're setting up new user testing groups.
  - Medi-Cal members shouldn't have to pay so they don't wait on hold, so we're having to figure out new business models.
- Mostly we feel like this (PUPPYYYY)
- Please reach out if you want to get involved in this work with us!
